src/styles/search.css

```diff
@@ -0,0 +1,27 @@
+.search-form {
+  display: flex;
+  flex-direction: row;
+  align-self: center;
+  justify-content: space-between;
+  margin-bottom: 10vh;
```

Comment:
This is a very odd unit to use here, vertical height makes no sense in this context. Should replace this with a unit like em ideally